app/views/billboards/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<% provide :head_tags do %>
+<meta name='turbolinks-visit-control' content='reload'>
+    <script>
+
+
+    document.addEventListener("DOMContentLoaded", function(){
```

Comment:
Don't write inline JS, extract it into `billboards.js`

Reply:
I spend a  lot of time trying to get rid of this inline JS but I couldn't resolve this problem